core/src/test/scala/tech/beshu/ror/unit/acl/logging/AuditingToolTests.scala

```diff
@@ -49,7 +49,8 @@ class AuditingToolTests extends AnyWordSpec with MockFactory {
         val auditingTool = new AuditingTool(
           AuditingTool.Settings(
             RorAuditIndexTemplate.from("'test_'yyyy-MM-dd").right.get,
-            new DefaultAuditLogSerializer
+            new DefaultAuditLogSerializer,
+            None
```

Comment:
are we able to test remote cluster here?

Reply:
Nope, we provide `mock[AuditSinkService]` to `AuditingTool`. Using audit cluster settings here is confusing as it's not used by `AuditingTool` at all, but at the layer above in `ReadonlyRest`. It's here, because we have to pass something to provide full audit settings. Maybe it means `AuditingTool.Settings` should be restructured, so we don't have to use cluster settings here...? Anyway, we have it covered in integration tests ;)